specification/communication/data-plane/CallAutomation/stable/2023-10-15/communicationservicescallautomation.json

```diff
@@ -0,0 +1,3007 @@
+        "playSources": {
+          "description": "The source of the audio to be played.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PlaySource"
+          }
+        },
+        "playTo": {
+          "description": "The list of call participants play provided audio to.\r\nPlays to everyone in the call when not provided.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommunicationIdentifierModel"
+          }
+        },
+        "playOptions": {
+          "$ref": "#/definitions/PlayOptions",
+          "description": "Defines options for playing the audio."
+        },
+        "operationContext": {
+          "description": "The value to identify context of the operation.",
+          "type": "string"
+        },
+        "operationCallbackUri": {
+          "description": "Set a callback URI that overrides the default callback URI set by CreateCall/AnswerCall for this operation.\r\nThis setup is per-action. If this is not set, the default callback URI set by CreateCall/AnswerCall will be used.",
+          "type": "string"
+        }
+      }
+    },
+    "PlaySourceType": {
+      "description": "Defines the type of the play source",
+      "enum": [
+        "file",
+        "text",
+        "ssml"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "PlaySourceType",
+        "modelAsString": true
+      }
+    },
+    "PlaySource": {
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/PlaySourceType"
+        },
+        "playSourceCacheId": {
+          "description": "Defines the identifier to be used for caching related media",
+          "type": "string"
+        },
+        "file": {
+          "$ref": "#/definitions/FileSource",
+          "description": "Defines the file source info to be used for play"
+        },
+        "text": {
+          "$ref": "#/definitions/TextSource",
+          "description": "Defines the text source info to be used for play"
+        },
+        "ssml": {
+          "$ref": "#/definitions/SsmlSource",
+          "description": "Defines the ssml(Speech Synthesis Markup Language) source info to be used for play"
+        }
+      }
+    },
+    "PlayOptions": {
+      "required": [
+        "loop"
+      ],
+      "type": "object",
+      "properties": {
+        "loop": {
+          "description": "The option to play the provided audio source in loop when set to true",
+          "type": "boolean"
+        }
+      }
+    },
+    "FileSource": {
+      "required": [
+        "uri"
+      ],
+      "type": "object",
+      "properties": {
+        "uri": {
+          "description": "Uri for the audio file to be played",
+          "type": "string"
+        }
+      }
+    },
+    "VoiceKind": {
+      "description": "Voice kind type",
+      "enum": [
+        "male",
+        "female"
+      ],
```

Comment:
Just for consideration. Some service may want to avoid male/female, instead using voice name like Sara etc.

Reply:
We have another property voiceName for it.